pkg/webhook/core.oam.dev/v1alpha2/application/validating_handler_test.go

```diff
@@ -164,8 +164,7 @@ var _ = Describe("Test Application Validator", func() {
 				},
 			},
 			Data: runtime.RawExtension{
-				Raw: []byte(`
-{"apiVersion":"core.oam.dev/v1beta1",
+				Raw: []byte(`{"apiVersion":"core.oam.dev/v1beta1",
```

Comment:
It's strange that this bug not reproducted in my  computor.   How did you find it? 

@Somefive

Reply:
I tried to recreate a new ControllerRevision and found the format matter. I guess the reason might be the first byte in Raw should not be \n otherwise something weird will happen.

Reply:
The APIServer returns confused message  ` INTERNAL_ERROR (0x2)`.  
Thanks for you let me  learn some new knowledge.